infra/modules/feature_flags/main.tf

```
@@ -0,0 +1,9 @@
+resource "aws_ssm_parameter" "feature_flags" {
+  # checkov:skip=CKV2_AWS_34:Feature flags values don't need to be encrypted
+  for_each = var.feature_flags
+
+  name        = "/service/${var.service_name}/${each.key}"
```

It might be nice to have the FF stuff partitioned out, something like:
```suggestion
  name        = "/service/${var.service_name}/feature-flag/${each.key}"
```
If folks name their flags `enable_` (or similar) then `/service/my-service-dev/enable-new-flow` isn't too bad, but `/service/my-service-dev/bar` would be more confusing than `service/my-service-dev/feature-flag/bar`.

---

Good idea